examples/pipelines/simple_pipeline/components/image_cropping/src/main.py

```diff
@@ -0,0 +1,78 @@
+"""
+This component crops images by removing the borders.
+"""
+import io
+import logging
+
+import numpy as np
+from PIL import Image
+import dask.dataframe as dd
+
+from fondant.component import TransformComponent
+from fondant.logger import configure_logging
+
+from image_crop import remove_borders
+
+configure_logging()
+logger = logging.getLogger(__name__)
+
+
+def extract_width(image_bytes: bytes) -> np.in16:
+    """Extract the width of an image
+
+    Args:
+        image_bytes (bytes): input image in bytes
+
+    Returns:
+        np.int16: width of the image
+    """
+    return np.int16(
+        Image.open(io.BytesIO(image_bytes)).size[0]
+    )
+
+
+def extract_height(image_bytes: bytes) -> np.int16:
+    """Extract the height of an image
+
+    Args:
+        image_bytes (bytes): input image in bytes
+
+    Returns:
+        np.int16: height of the image
+    """
+    return np.int16(
+        Image.open(io.BytesIO(image_bytes)).size[1]
+    )
```

Comment:
Can we combine these so we don't need to open the image twice?

We could also expect these as part of the input dataset, but I guess that would make this component less reusable, since then it would only work on datasets where this information is already available.